### ISSUE TITLE
Improve highlighting for interpolated strings containing hyperlinks

### ISSUE THREAD

**Issue by Suchiman** [dotnet/roslyn#5863]
![2015-10-11_01-34-32](https://cloud.githubusercontent.com/assets/7110884/10413813/5a59515a-6fb8-11e5-892a-75a04c18548c.png)
When you have a string with a hyperlink in it, the blue will override all regular highlighting colors as it appears to.

**Comment by paul1956** [comment]
The blue isn't the real problem. Visual Studio lets you click and go to the URL which makes no spence if there are substituations. If they fix that the blue should just disappear.

**Comment by Suchiman** [comment]
@paul1956 indeed that's more reasonable

**Comment by CyrusNajmabadi** [comment]
Hyperlink highlighting is not handled by Roslyn.  It's a global editor feature.  You can enable/disable the feature here: 
![image](https://cloud.githubusercontent.com/assets/4564579/10420460/6430958a-7049-11e5-90aa-4c7517f22673.png)

**Comment by Suchiman** [comment]
@CyrusNajmabadi interesting, i didn't know that.
For me that's fine since i didn't ever really use that feature.
Should this be closed then as "By Design", "Won't fix" or do you have something in mind for those who do care?

**Comment by CyrusNajmabadi** [comment]
No, I wouldn't describe it as 'by design'.  It's definitely a bug IMO.  It's just a bug due to the feature being implemented in a rather simple manner by the editor team (which I don't blame them for).  99.9% of the time their approach will work.  Occasionally it's wrong.  Ideally the Editor would work with the Language APIs to try to get it right 100% of the time.

That said, this bug is just about the most minor thing I've ever seen.  And pretty much everyone has ***much*** more important stuff on their plate.  So I could imagine this just never getting fixed.  That meets my bar for closing.  (But I also don't mind real bugs staying open, even if they're very low pri).

I would say that this should go to the Editor.  However, I don't know the best way for you to move bugs to them.  So just leave it on us, and hopefully our triage people will know what to do.

**Comment by paul1956** [comment]
I think the bug should be fixed or even better provide an option or just never enable One-Click URL navigation for interpolated strings. I find for strings this is a useful feature that I would not want to disable.

**Comment by Pilchie** [comment]
Tagging @olegtk and @markw-t.  I thought we'd had a previous issue about this tagger, but I can't find it right now.

**Comment by paul1956** [comment]
It was issue #5041

**Comment by Pilchie** [comment]
Thanks @paul1956

**Comment by niico100** [comment]
Colouring white is way more intuitive, it should take priority

**Comment by CyrusNajmabadi** [comment]
Closing out as external.  Fix would need to be somewhere in VS, not roslyn.
